Add nebari typer cli changes (#150)
nebari-dev/nebari-docs: docs/docs/getting-started/installing-nebari.md

:::

---
## What's next?

### Nebari Init and Guided Init

> Nebari Init create and initialize your Nebari configuration. Guided init does the same but user can have different experience while running it (more enhanced).

Reviewer: ```suggestion
Nebari Init creates and initializes your Nebari configuration. Guided init does the same but you can have a step-by-step experience while running it.
```

You can pass the `--help` flag to the `init` command to check all the arguments and options available for it.

```bash
nebari init --help 
```

![A representation of the output generated when Nebari init help command is executed, the output contains a list of the available options and arguments and their use.](/img/getting-started/nebari-init-help-2.png "Nebari's init help command line output")

:::tip

> You can pass the `--help` flag with the `--guided-init` flag to the `init` command to check all the arguments and options available for it.

Reviewer: Do we mean `nebari init --help --guided-init` here? This command starts the guided init wizard for me right, I'm not sure if that's intentional or a bug.

If it's expected, I think we may need to update this tip message.

Author: Yes, you are right. I have added the reference to the guided-init command in the tip section.

Reviewer: Thanks!

:::

---
## What's next?

### Nebari Init and Guided Init

You can run help command for init to check for all the arguments and options available for it.

```bash
nebari init --help
```

![A representation of the output generated when Nebari init help command is executed, the output contains a list of the available options and arguments and their use.](/img/getting-started/nebari-init-help.png "Nebari's init help command line output")

> If you are new to Nebari you can run the nebari guided-init help command as the init command option and see all the options available to it.

Reviewer 2: ```suggestion
If you are new to Nebari you can run the nebari `guided-init` with the `--help` command to see all the arguments and options available.
```
